MerlinAU.sh

```diff
@@ -3985,18 +4018,29 @@ _IncrementDay_()
     echo "$day $month $year"
 }
 
-##----------------------------------------##
-## Modified by Martinski W. [2024-May-18] ##
-##----------------------------------------##
+##------------------------------------------##
+## Modified by ExtremeFiretop [2024-Nov-26] ##
+##------------------------------------------##
 matches_day_of_month()
 {
     local curr_dom="$1"
     local dom_expr="$2"
-    local domStart  domEnd
+    local domStart domEnd expanded_days
 
     if [ "$dom_expr" = "*" ]
     then  # Matches any day of the month #
         return 0
+    elif echo "$dom_expr" | grep -q '/'
+    then
```

Comment:
Ah, so the specific cases for "day intervals" were missing altogether. LOL!!!
Like pretty much all users, we don't set up the cron schedule with day intervals within the week or the month so never noticed at all that the cases were missing, but somebody out there will try someday so now they will work fine!!
Good job, bud!!

Reply:
@Martinski4GitHub 

Exactly! A very clear oversight on my part, but a very great catch on your part!